examples/thermostat/efr32/include/ThermostatUI.h

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "ThermostatIcons.h"
+#include "glib.h"
+#include "lcd.h"
+
+class ThermostatUI
```

Comment:
Given that these are all static members, should this be a namespace instead?

Reply:
I kept this one as a class as we might add more stuff to ThermostatUI in the near future